README.md

```diff
@@ -93,6 +93,22 @@ Diagrams are generated "automatically" by ???
 
 ## User Guide
 
+###Database Management
```

Comment:
need a space between thee ### tag and the title text

Reply:
Done, nice catch